pages/cloud/private-cloud/lifecycle_policy/guide.de-de.md

```diff
@@ -39,6 +39,8 @@ Hosted Private Cloud
 * DC2014 & DC2016 (AMD PileDriver)
 * SDDC2014 & SDDC2016 (Intel Ivy Bridge, Intel Haswell)
 * SDDC2018 (Intel Broadwell, Intel Haswell)
+* Essentials (Intel Broadwell)
```

Comment:
Hello @Hubmar, you added these 2 lines on the de-de sub but not on the others. Should I replicate them onto all subs ?

Reply:
I decided to replicate the lines.